databricks_cli/workspace/cli.py

```diff
@@ -110,6 +110,11 @@ def export_workspace_cli(source_path, target_path, format, overwrite): # NOQA
     format is documented at
     https://docs.databricks.com/api/latest/workspace.html#notebookexportformat.
     """
+    if os.path.isdir(target_path):
+        file_info = get_status(source_path)
+        assert file_info.is_notebook, 'Export can only be called on a notebook'
```

Comment:
This should always lead to a runtime error, but `assert` is not guaranteed to run.